tests/test_container_utils.py

```diff
@@ -0,0 +1,48 @@
+from pathlib import Path
+
+from pytest_mock import MockerFixture
+
+from dangerzone.container_utils import Runtime
+from dangerzone.settings import Settings
+
+
+def test_get_runtime_name_from_settings(mocker: MockerFixture, tmp_path: Path) -> None:
+    mocker.patch("dangerzone.settings.get_config_dir", return_value=tmp_path)
+
+    settings = Settings()
+    settings.set(
+        "container_runtime", "/opt/somewhere/new-kid-on-the-block", autosave=True
+    )
+
+    assert Runtime().name == "new-kid-on-the-block"
+
+
+def test_get_runtime_name_linux(mocker: MockerFixture, tmp_path: Path) -> None:
```

Comment:
Why not make this a test that runs only on Linux platforms, and thus remove the need for mocking? Similarly for the non-Linux test below.

Reply:
Because it makes it possible to catch the errors on my dev machine :-)

Reply:
Heh, I assumed as much. This makes the test cases a bit more synthetic, but I do get that we'll catch errors more easily, so it's a pass from me. Resolving this thread...